site/content/blog/2019-10-21-localizing-matterpoll/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Localizing Matterpoll"
+slug: localizing-matterpoll
+date: 2019-11-05T10:49:35+02:00
+categories:
+    - "plugins"
+    - "go"
+author: Ben Schumacher
+github: hanzei
+community: hanzei
+---
+
+[Matterpoll](https://github.com/matterpoll/matterpoll) is a plugin that allows users to create polls in Mattermost. Since Mattermost is [localized in 16 different languages](https://docs.mattermost.com/developer/localization.html), it’s optimal that Matterpoll is similarly localized.
+
+Because we rely on contributors to do the translations, we want to make it easy for them to translate new strings and determine if already translated strings needs to be updated because the "source" text changed. On the other hand, Matterpoll only has two maintainer ([@kaakaa](https://github.com/kaakaa) and [me](https://github.com/hanzei)) and no infrastructure of its own to work with. Using a translation server like [Transifex](https://en.wikipedia.org/wiki/Transifex) or [Weblate](https://en.wikipedia.org/wiki/Weblate) is not an option.
```

Comment:
```suggestion
Because we rely on contributors to do the translations, we want to make it easy for them to translate new strings and determine whether already translated strings need to be updated because the "source" text changed. On the other hand, Matterpoll only has two maintainers ([@kaakaa](https://github.com/kaakaa) and [me](https://github.com/hanzei)) and no infrastructure of its own to work with. Using a translation server like [Transifex](https://en.wikipedia.org/wiki/Transifex) or [Weblate](https://en.wikipedia.org/wiki/Weblate) is not an option.
```